### PR TITLE
feat: Add dataset creator

### LINKED ISSUE #11: Add publisher to catalog
We need at least an identifier (or abbreviation) for displaying in the search results and demonstrator UI.

### PR DESCRIPTION
**Pull request by ddeboer**
Fix #11.

### CODE REVIEW

File: catalog/datasets/aat.jsonld [hunk 1]
```diff
@@ -8,6 +8,11 @@
       "@value": "Art & Architecture Thesaurus (AAT)"
     }
   ],
+  "creator": [
```
Comment (ddeboer) [inline catalog/datasets/aat.jsonld:11]: I used `schema:creator` here, but would be more than happy to discuss whether `schema:publisher` is more appropriate.

File: src/catalog.ts [hunk 2]
```diff
@@ -25,8 +26,10 @@ export class Catalog {
       PREFIX schema: <http://schema.org/>
         SELECT * WHERE {
           ?dataset a schema:Dataset ;
-            schema:distribution ?distribution ;
-            schema:name ?name .
+            schema:name ?name ;
+            schema:creator ?creator ;
+            schema:distribution ?distribution .
+          ?creator schema:identifier ?creatorIdentifier .
```
Comment (ddeboer) [inline src/catalog.ts:32]: To keep things simple, I support only a single creator for now.

File: src/catalog.ts [hunk 3]
```diff
@@ -48,6 +51,12 @@ export class Catalog {
           new Dataset(
             new IRI(bindings.get('?dataset').value),
             bindings.get('?name').value,
+            [
+              new Organization(
```
Comment (ddeboer) [inline src/catalog.ts:55]: Also up for debate: are organizations the only creators or can individual persons be creating agents as well? For now, we only have organizations.

**Comment by ddeboer** [comment]
@sdevalk Please also have a look at the IRIs I used for the creators/publishers: do they make sense?

File: catalog/datasets/aat.jsonld (hunk 1 above)
Comment (sdevalk) [reply catalog/datasets/aat.jsonld:11]: Agree!

File: catalog/publishers.jsonld [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      "@id": "http://www.getty.edu/research/",
+      "identifier": "Getty"
+    },
+    {
+      "@id": "https://www.cultureelerfgoed.nl",
+      "identifier": "RCE"
+    },
+    {
+      "@id": "https://www.wereldculturen.nl",
+      "identifier": "Getty"
```
Comment (sdevalk) [inline catalog/publishers.jsonld:14]: I think 'Getty' should be replaced by 'NMvW' (or 'Wereldculturen' - its abbreviation may not be known by people)?

File: src/catalog.ts (hunk 2 above)
Comment (sdevalk) [reply src/catalog.ts:32]: Makes sense! We'll deal with multiple creators as soon as a relevant source arrives.

File: src/catalog.ts (hunk 3 above)
Comment (sdevalk) [reply src/catalog.ts:55]: Agree!

File: catalog/publishers.jsonld (hunk 4 above)
Comment (ddeboer) [reply catalog/publishers.jsonld]: Decided to use NMVW (capital ‘V’ as that’s [how they write it themselves](https://www.tropenmuseum.nl/nl/over-het-tropenmuseum/stichting-nationaal-museum-van-wereldculturen)) for now. For API users, we have the IRI for dereferencing (although it resolves to HTML not RDF in most cases) and finding the name and details of the organization. Something similar may be nice in the demonstrator as well, but that’s out of scope for now.

**Review by sdevalk — APPROVED** [review]

**Pull request merged**